Web/Design-Pattern-2/Assignment-2/src/ChainOfResponsiblityinterface.java

```diff
@@ -0,0 +1,20 @@
+import java.util.Scanner;
+
+public class ChainOfResponsiblityinterface {
```

Comment:
ACTION: Use Pascal casecading for class name so it should be "ChainOfResponsiblityInterface"